src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
+@import './utils/variables';
+@import './blocks/card';
+@import './blocks/header';
+@import './blocks/stars';
+
 body {
   margin: 0;
+  padding: 0;
+
+  font-family: $font-family;
```

Comment:
Ensure that the variable `$font-family` is defined in the `variables` file being imported. This will prevent any potential issues with undefined variables.